examples/express_example.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const epsagon = require('epsagon');
+
+epsagon.init({
+    token: 'my-secret-token',
+    appName: 'my-app-name',
+    metadataOnly: false,
+});
+
+const app = express()
+
+app.get('/', (req, res) => res.send('Hello World!'))
```

Comment:
It's not a good idea to add the wrapper implicitly. Sometimes the user will have an express application with wrappers in other places. And it makes this library much complicated (needing to make sure a wrapper is not added twice)